client/my-sites/stats/stats-date-control/stats-date-control-picker-shortcuts.tsx

```diff
@@ -1,4 +1,5 @@
 import { Button } from '@wordpress/components';
+import { Icon, check } from '@wordpress/icons';
```

Comment:
Can you use icons from `@wordpress/icons`? This will help to move away from Gridicons and reduce one of Odyssey's dependencies in the future.

Reply:
This is correct. I only noticed the `gridicon` CSS class.